src/main/java/org/opentripplanner/routing/graphfinder/StopAtDistance.java

```diff
@@ -55,6 +57,15 @@ public int compareTo(StopAtDistance that) {
     return (int) (this.distance) - (int) (that.distance);
   }
 
+  public AccessEgress toAccessEgress(StopIndexForRaptor stopIndex, boolean fromTarget) {
```

Comment:
Move the mapping to a Mapper class under the RoutingWorker, and rename this class to `NearbyStop`.